### ISSUE TITLE
No internet access

### ISSUE THREAD

**Issue by machadogj** [weavenet/aws-ec2-vpn#4]
Hello, first, thanks for the awesome project. I followed the instructions, and was able to successfully connect to the VPN with "send all traffic over vpn connection", however as soon as I connect, I loose all connection to internet.

Also, after several connects/disconnects the server stops responding (ie: cannot connect anymore) and I have to reboot the server from the EC2 console.

What is the best way to troubleshoot this? How could I ssh into the instance?

**Comment by nickoneill** [comment]
Yeah same here. I set this up two days ago and it was working great, but now when I successfully connect, I get no connection to the internet. I wonder if everything comes back up normally after a reboot?

**Comment by weavenet** [comment]
@nickoneill @machadogj Thanks for the feedback.  I previously saw the connection die and not be able to reconnect, however have not been able to dig in and troubleshoot as it was sporadic.

I have a branch that allows for SSH access to the instance (I will probably merge this into master for easier troubleshooting) if you want to add a SSH key to the instance and connect to look at error logs (username ubuntu).

https://github.com/weavenet/aws-ec2-vpn/blob/ssh/aws-ec2-vpn.json

Thanks and let me know what you find.

**Comment by nickoneill** [comment]
Is there a straightforward way to apply the ssh config to the current cloud formation? Or should I delete and create a new one?

**Comment by weavenet** [comment]
Not sure, but probably easiest to recreate it.

**Comment by nickoneill** [comment]
I recreated with the ssh settings (a few other tweaks were required to pass the key name through the setup script), again VPN worked great for a few connections and then stopped accepting new connections (perhaps this can be mitigated by manually disconnecting rather than sleeping my laptop, but regardless...). I rebooted the ec2 instance via the aws console and now I can connect again but no outbound internet access.

Since I have a reproducible case right in front of me and can SSH into it - any idea where I can look around to pinpoint the issue?

**Comment by weavenet** [comment]
Hi nick,

I’d recommend you check the various logs in /var/log for errors, timeouts, etc.  defiantly messages and also see if the VPN service has a separate log. Let us know what you find.
